ops-catalog/schema-inference.flow.yaml

```diff
@@ -0,0 +1,36 @@
+collections:
+  estuary/global/schema-inference-L2:
+    schema: schema-inference.schema.yaml
+    key: [/collection_name]
+    projections:
+      collection_name: /collection_name
+      schema: /schema
+
+    derive:
+      using:
+        sqlite: {}
+      # The idea with this L2 collection is to merge
+      # all data planes' L1 collections together
+      transforms:
+        - name: from-ops.us-central1.v1
+          source: ops.us-central1.v1/private/schema-inference-L1
+          shuffle: any
+          lambda: select $collection_name, $schema;
+  ops.us-central1.v1/private/schema-inference-L1:
```

Comment:
nit: IMO a better name would be `ops.us-central1.v1/inferred-schemas/L1`. `/private/` feels like a bit of a smell, since visibility is already handled by grants.

Reply:
@psFried FYI, I'm deeply regretting that we placed `logs` and `stats` at the tenant top-level...

Reply:
@jgraettinger I'm curious why? Is it making it more difficult to manage access grants?

Reply:
If I want to grant `aliceCo` access to `bobCo/logs`, I have to give them a read grant to all of `bobCo/` since grants operate on prefixes.